common-tasks/tips-and-tricks.md

```diff
@@ -58,4 +58,5 @@ Trim for standalone AppVMs
 ---------------------
 The `qvm-trim-template` command is not available for a standalone AppVM.
 
-It is still possible to trim the AppVM disks by using the `fstrim --all` command from the appvm
+It is still possible to trim the AppVM disks by using the `fstrim --all` command from the appvm.
+Note you may need to add the `discard` option to the mount line in `/etc/fstab` inside the standalone AppVM before this will work as expected.
```

Comment:
Those two things are not related in the way you've described:
 - `fstrim` manually send discard requests for unused space
 - `discard` option automatically send discard requests whenever space is freed (file delete, etc)